server/supernova/sc/sc_osc_handler.hpp

```diff
@@ -325,6 +327,10 @@ class sc_osc_handler : private detail::network_thread, public sc_notify_observer
                                  AsyncStageFn stage3, AsyncStageFn stage4, AsyncFreeFn cleanup, int completionMsgSize,
                                  void* completionMsgData);
 
+    void send_message_from_RT(World* world, FifoMsg& msg);
+
+    void send_message_to_RT(World* world, FifoMsg& msg);
```

Comment:
these can both be `const` members

Reply:
they could, but for consistency we would have to do this for many other member functions and I think this is out of scope for this PR.

Reply:
no, you can change it now, some code being const correct is better than none.

Reply:
this is a typical problem working on this codebase and on legacy and underreviewed codebases in general. many good changes would be held back in the name of "consistency", but modernizing the code in a consistent manner is psychically draining. so, a better strategy is to make incremental improvements and, to the extent that it's possible, ensure that newly added code meets new standards of quality.

Reply:
ok